plugins/github/server/api/github.ts

```diff
@@ -65,13 +66,14 @@ router.get(
       }
     }
 
-    const githubUser = new GitHubUser({ code: code!, state: teamId });
+    const client = await GitHub.authenticateAsUser(code!, teamId);
+    const installationsByUser = await GitHub.requestAppInstallations(client);
```

Comment:
Isn't this less performant than before? Why change to loading all installations 🤔

Reply:
We were always loading all the installations. There doesn't seem to be an API to load a single installation for a given user.